sw/device/lib/base/testing/mock_mmio_test_utils.h

```diff
@@ -0,0 +1,134 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_BASE_TESTING_MOCK_MMIO_TEST_UTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_BASE_TESTING_MOCK_MMIO_TEST_UTILS_H_
+
+#include <initializer_list>
+#include <memory>
+#include <random>
+#include <stdint.h>
+#include <string.h>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/mmio.h"
+
+namespace mock_mmio_utils {
```

Comment:
TBH I'd just leave this namespace as mock_mmio.

Reply:
Sounds good. I will update this.

Reply:
Done.